tests/Unit/ParallelAlgorithms/Events/Test_ObserveFields.cpp

```diff
@@ -256,7 +254,7 @@ struct ComplicatedSystem {
       check_component("Error(Tensor2)_yx", TensorVar2{}, 0, 1);
       check_component("Error(Tensor2)_yy", TensorVar2{}, 1, 1);
     }
-
+    // NOLINTNEXTLINE
```

Comment:
If these do end up needed, can you `NOLINTNEXTLINE(error...)` to only NOLINT the specific errors? Also a few lines above